docs/content/error/$compile/ctxoverride.ngdoc

```diff
@@ -0,0 +1,13 @@
+@ngdoc error
+@name $compile:ctxoverride
+@fullName DOM Property Security Context Overrid
+@description
+
+This error occurs when the security context for a property is defined via {@link ng.$compilerProvider#addPropertySecurityContext addPropertySecurityContext()} multiple times under different security contexts.
```

Comment:
This link is incorrect

Reply:
That's what you get for blindly copy-pasting my suggestions 😛

Reply:
Still not fixed :<